app/forms/date_form.py

```diff
@@ -282,7 +284,9 @@ def get_referenced_offset_value(answer_min_or_max, answer_store, metadata):
     elif 'meta' in answer_min_or_max:
         value = get_metadata_value(metadata, answer_min_or_max['meta'])
     elif 'answer_id' in answer_min_or_max:
-        value = get_answer_store_value(answer_min_or_max['answer_id'], answer_store, group_instance=0)
+        session_data = get_session_store().session_data
+        schema = load_schema_from_session_data(session_data)
```

Comment:
At this point you already have metadata, therefore, you can make use of `load_schema_from_metadata` instead of getting the session from globals?.

Reply:
Thanks! That makes things a little nicer.